File: MainApp/src/main/java/com/sleticalboy/learning/bt/ble/IConnectCallback.java
package com.sleticalboy.learning.bt.ble;

/**
 * Created on 20-8-13.
 *
 * @author Ben binli@grandstream.cn
 */
public interface IConnectCallback {

    void onFailure(Connection connection, BleException e);

    void onSuccess(Connection connection);
}

File: MainApp/src/main/java/com/sleticalboy/learning/bt/ble/IConnectCallback.kt
package com.binlee.learning.bt.ble

/**
 * Created on 20-8-13.
 *
 * @author Ben binli@grandstream.cn
 */
interface IConnectCallback {

  fun onFailure(connection: Connection, e: BleException)
  fun onSuccess(connection: Connection)
}